### ISSUE TITLE
Interactive Terminal: Undo Latest Update

### ISSUE THREAD

**Issue by MarkMoretto** [microsoft/vscode#142345]
In the latest iteration of VSCode, the terminal and side panel just...don't work.  Like, I have no idea what y'all did, but exiting the interactive terminal doesn't even close the side panel.  Why is this a good idea?

When operational, the Terminal panel, which no longer has an icon/tab at the top of the side panel, has a downward-facing caret.  When typing Exit into the terminal, the terminal interface disappears, but the downward caret stays.  Clicking on the terminal bar causes the terminal and Problems (why?) pane to collapse 

My info:

+ Version: 1.64.0
+ Commit: 5554b12acf27056905806867f251c859323ff7e9
+ Date: 2022-02-03T04:23:11.224Z
+ Electron: 13.5.2
+ Chromium: 91.0.4472.164
+ Node.js: 14.16.0
+ V8: 9.1.269.39-electron.0
+ OS: Linux x64 5.10.0-5mx-amd64

Screenshots of the evolution:

Terminal open to prompt:  
![vscode-update-terminal-1](https://user-images.githubusercontent.com/25012707/152697894-e9294906-719a-42a9-9ac0-cb3cbac7980b.png)

After typing `exit` and pressing `Enter`:  
![vscode-update-terminal-2](https://user-images.githubusercontent.com/25012707/152697895-db798a00-1978-4e1b-a3ac-75076626f848.png)

After clicking on Terminal bar:  
![vscode-update-terminal-3](https://user-images.githubusercontent.com/25012707/152697897-52d32f6c-65a9-47d6-8920-908be0da1b79.png)

**Comment by sbatten** [comment]
@MarkMoretto sorry for the confusion here. It looks like the terminal is placed within the problems tab. is this expected for you? if not, can you please try: Right click the terminal icon and click "reset location"

**Comment by MarkMoretto** [comment]
Hi,  thanks for the response!

It seems to be alright today.  i have the keystroke `CTRL + j` set to launch the terminal and typing exit appears to work.  I also launch VSCode with the `--disable-gpu` flag, but that doesn't seem to impact the platform.

As of now, I have successfully move the side panel/terminal window from the bottom to the right side of the window via drag-and-drop, but there is only a terminal icon at the top of the side panel.

When right-clicking the terminal icon, there's options to Reset Terminal and Hide Terminal, but nothing about moving it back to the bottom of the window.

The trash can icon is there, the plus sign (+) select menu is there with different terminals, and clicking the name of the current terminal brings up a menu with Change icon..., Split terminal,  Change color..., etc.

tl;dr - Terminal seems okay for now, but options to move the panel _might_ be missing.

**Comment by sbatten** [comment]
"Reset Location" will move it to the bottom of the window. Additionally "Move Views to Panel" moves them to the bottom panel.

**Comment by MarkMoretto** [comment]
I mean, with all due respect, why is it this complicated?  It worked fine before.

**Comment by sbatten** [comment]
I'm having trouble understanding what the current difficulties you are having are. The screenshots you shared show that the terminal was nested inside the problems views which seemed to be the original problem but was not a choice we made. If it happened unintentionally, its possible there is a bug, or you moved it by mistake and the behavior was not what you expect. I would like to help you get to the state you like with the functionality we provide. Screen shots and descriptions are helpful. again, I am sorry for the confusion.
